spec/keytar-spec.js

```diff
@@ -0,0 +1,130 @@
+var assert = require('chai').assert
```

Comment:
Yeah, it's quite minor honestly. Given we'd pay for it only in tests, I would still suggest doing it so that we can get rid of `async.waterfall` and use a modern syntax across the entire Atom org.